docs/developers_guide.rst

```diff
@@ -109,6 +120,11 @@ The simplest way to debug ORA2 code is with PDB - Python's built in debugger.
 One caveat: Since ORA2 has code that executes either in studio or lms context, one must be attached to the corret shell 
 in order for the breakpoints to be hit.
 
+Debugging JavaScript
+------------------
+Adding ``debugger`` to the JavaScript should work for ``development build`` and ``development server``.
+It should not work in ``production build`` however, adding breakpoint in mapped files for debugging should work on both build.
+
```

Comment:
Final proposed wording change (with @leangseu-edx to fill in locations from browser debug tools)

```suggestion
Debugging JavaScript
------------------
For debugging JS in Devstack, first follow the instructions for "Hot Reload JS". This enables source maps and allows for placing breakpoints in source-mapped files from the browser dev tools.

 Locate code by browsing to <Leangseu, add path here> and breakpoints should toggle with hot-reloading.

```